### PR TITLE
CORE-20695: Retire `v1` REST version

### PR DESCRIPTION
**Pull request by BenYip123**
- Retire v1 REST version
- Remove usages of v1 endpoints
- Bump up versions inside some tests
- Remove deprecation tags where it is no longer applicable
- Remove swaggerBaseline-v1

E2E tests: https://github.com/corda/corda-e2e-tests/pull/656

### CODE REVIEW

File: processors/rest-processor/src/integrationTest/resources/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Open API compatibility testing
 
-File [swaggerBaseline-$version.json](./swaggerBaseline-v1.json) represents a snapshot of Open API which REST Worker currently
+File [swaggerBaseline-$version.json](./swaggerBaseline-v5_1.json) represents a snapshot of Open API which REST Worker currently
```
Comment (vkolomeyko) [inline processors/rest-processor/src/integrationTest/resources/readme.md]:
```suggestion
File [swaggerBaseline-$version.json](./swaggerBaseline-v5_3.json) represents a snapshot of Open API which REST Worker currently
```

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Review by anton-subbotin — COMMENTED** [review]
There still are usages of `/v1/` paths across the codebase, including under:
- `tools/corda-runtime-gradle-plugin/` - where applicable, should be switched to use earliest supported version, e.g. `v5_1`, @tlawson3 please comment
- `libs/permissions/` - mentioned in tests
- `libs/corda-sdk/` - affects Corda CLI (`bootstrap rbac` command)

**Review by anton-subbotin — CHANGES_REQUESTED** [review]
There still are usages of `/v1/` paths across the codebase

**Comment by tlawson3** [comment]
> There still are usages of `/v1/` paths across the codebase, including under:
> 
> * `tools/corda-runtime-gradle-plugin/` - where applicable, should be switched to use earliest supported version, e.g. `v5_1`, @tlawson3 please comment
> * `libs/permissions/` - mentioned in tests
> * `libs/corda-sdk/` - affects Corda CLI (`bootstrap rbac` command)

Regarding the gradle plugin

- net/corda/gradle/plugin/cordalifecycle/EnvironmentSetupHelper.kt Updating to 5_1 should be fine. I've created task https://r3-cev.atlassian.net/browse/ES-2460 to refactor it to remove the Unirest use but that would be outside the scope of this PR.
- net/corda/gradle/plugin/ProjectUtils.kt fun getExistingNodes isn't used so feel free to delete the function

**Review by anton-subbotin — DISMISSED** [review]
LGTM

**Comment by vkolomeyko** [comment]
Could we also remove source file `testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/websocket/client/SmokeTestWebsocketClient.kt`? As it contains reference to `api/v1` and I do not think this Websocket functionality is still used in E2E tests.

In fact we should be able to remove the whole package: `net.corda.e2etest.utilities.websocket.client`.

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**